misaligned_fixed&Test.py

```diff
@@ -0,0 +1,31 @@
+
+def print_color_map():
+    major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
+    minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
+    list = []
+
+    for i, major in enumerate(major_colors):
+        for j, minor in enumerate(minor_colors, 1):
+            interval = i * 5 + j
+            list.append(interval)
+            #assert list[0] == 0
+            #print(f'{interval} | {major} | {minor}')

```

Comment:
Don't leave commented code in commits. Delete them.
If you need older code, git will keep them in the previous commit